### ISSUE TITLE
Grid freeze

### ISSUE THREAD

**Issue by fluxxion82** [monome/ansible#26]
When using the 128 varibright grid with a Switch connected to Ansible(latest firmware v1.5.0), probably within 10 min of use, the grid freezes up. Sometimes I can still see the pulse going through all the steps, and other times there's no pulse movement. Every time though, I can not move to any of the menu items or change any params in the current menu on the grid; I can't switch tracks, and go to the notes or durations, etc. Ansible continues to play though. In order to get the grid back to normal, (if using an external clock, I have to stop the external clock), using the switch module I switch to the other output, which currently isn't being used, then switch back to the Ansible connection. Once I do those steps, the grid starts to function normally. Haven't tried other firmware versions as I just bought the Ansible.

**Comment by tehn** [comment]
though you haven't tried other versions, have you reflashed the firmware to
confirm the newest is present?

**Comment by fluxxion82** [comment]
Yeah, I reflashed the firmware again and so far everything is looking good; the grid hasn't frozen up since. I'll close the issue. Thanks.

**Comment by fluxxion82** [comment]
Actually, I take that back..it just froze on me again. I feel like maybe having an external clock might be playing a part. I'll install the previous version and see if I have any issues.

**Comment by samdoshi** [comment]
Sounds like it relates to this bug: https://llllllll.co/t/is-it-normal-for-ansible-to-freeze-after-a-lot-of-arc-activity/4735

**Comment by fluxxion82** [comment]
Interesting. Definitely sounds like something is going on. I just posted a quick blurb there. I'm really feeling like, for me at least, having an external clock is triggering the issue. Last night I was going to start trying other firmware versions but decided to reflash the latest firmware and see how long it runs without freezing without an external clock, and so far it's still going. I switched the grid off while I was asleep and at work, but switched the grid back to Ansible and it's still going. I'll play around with it for a couple hours and if it's still going, I'll hook up the external clock and see how long it takes.
